util/rpcclient/rpcclient.go

```diff
@@ -256,9 +260,9 @@ func (c *RpcClient) Start(ctx_in context.Context) error {
 		var err error
 		var client *rpc.Client
 		if jwt == nil {
-			client, err = rpc.DialContext(ctx, url)
+			client, err = rpc.DialOptions(ctx, url, rpc.WithWebsocketMessageSizeLimit(c.config().WebsocketMessageSizeLimit))
```

Comment:
let's test here if WebsocketMessageSizeLimit == 0 (it can happen e.g. for validation-server-configs). If it's 0 either use default or don't apply rpc.WithWebsocketMessageSizeLimit

Reply:
0 will use default https://github.com/OffchainLabs/go-ethereum/blob/master/rpc/websocket.go#L254

(see also 
https://github.com/OffchainLabs/go-ethereum/blob/master/rpc/client.go#L216
https://github.com/OffchainLabs/go-ethereum/blob/master/rpc/client_opt.go#L72)